### ISSUE TITLE
Cannot load license Error: Invalid JWT

### ISSUE THREAD

**Issue by ThomasFrisendal** [neo4j-apps/neo4j-desktop#57]
After having upgraded to Mac OS X Mojave 10.14.3 on a MacBook Air I chose to update the Neo4j desktop browser as well. I got it downloaded, but when I open the Desktop app, there is a fast flash of a box saying something about internet connection. After that the screen below appears.  
![billede](https://user-images.githubusercontent.com/12172003/55015177-a4f66c80-4fec-11e9-9e0f-a2535e5f1581.png)
The message above was found in the log.log.
I am stuck. Cannot set offline mode.

**Comment by ThomasFrisendal** [comment]
Followed an advise from ameyrasoft in the community. Moved licenses.json somehwere else and tried again. Now it gives me the possibility of logging in using social media or wait 30 days. 
Why cannot I continue to use my 0Auth login? (Based on my email).

**Comment by oskarhane** [comment]
We have had some issues with the email registration and are re-working those parts right now.
Sorry for the inconvenience. 
You can continue to skip the registration multiple times as a workaround until we have fixed this issue.

**Comment by ProductKnowledgeGraph** [comment]
I'm also getting this same error. Can some please assist? I don't have the context of the community help referenced above nor can I locate it.

<img width="553" alt="Screen Shot 2019-04-28 at 11 45 51 AM" src="https://user-images.githubusercontent.com/48023836/56866772-5c472000-69ab-11e9-8e31-20d08638d113.png">

**Comment by polley** [comment]
Anyone have any solutions to this? Stuck in the same situation

**Comment by ThomasFrisendal** [comment]
@ProductKnowledgeGraph I continued to follow  @oskarhane 's advice above for some weeks. I did get an access key by using my Gituhb account. I learned, later that Neo4j stores passwords in a keychain. I had two of those. After that had been cleaned up (by Apple I suspect), I had to redo the database passwords (under Administration in the database's menu in Desktop). Along the road I also updated Desktop to the latest one. I am doing just fine now.
